### PR TITLE
[JIRA: HCPSDKFIORIUIKIT-1897] Signature Form Cell 

### PR DESCRIPTION
**Pull request by fredsap**

### CODE REVIEW

**Comment by MarcoEidinger** [comment]
Hi @fredsap ,

please
- sign CLA
- fix SwiftLint errors by running `swiftlint`
- fix formatting errors by running `swiftformat` for the newly introduced source code (causing `PR / ReuseComplianceCheck (pull_request)` to fail )
- remove app-preview.mp4 (causing `PR / ReuseComplianceCheck (pull_request)` to fail )

@billzhou0223 please do initial code review. In this current state (texts not localized, no configuration options, not adhering to our SwiftUI migration guidelines) this looks not ready for me to be merged into `main` branch

FYI: current snapshot tests fail (`PR / SnapshotTests (pull_request)`) unrelated to your changes and should be fixed once https://github.com/SAP/cloud-sdk-ios-fiori-snapshot-references/pull/21 gets merged.

**Comment by billzhou0223** [comment]
> Hi @fredsap ,
> 
> please
> 
> * sign CLA
> * fix SwiftLint errors by running `swiftlint`
> * fix formatting errors by running `swiftformat` for the newly introduced source code (causing `PR / ReuseComplianceCheck (pull_request)` to fail )
> * remove app-preview.mp4 (causing `PR / ReuseComplianceCheck (pull_request)` to fail )
> 
> @billzhou0223 please do initial code review. In this current state (texts not localized, no configuration options, not adhering to our SwiftUI migration guidelines) this looks not ready for me to be merged into `main` branch
> 
> FYI: current snapshot tests fail (`PR / SnapshotTests (pull_request)`) unrelated to your changes and should be fixed once [SAP/cloud-sdk-ios-fiori-snapshot-references#21](https://github.com/SAP/cloud-sdk-ios-fiori-snapshot-references/pull/21) gets merged.

Yes it needs to be polished before being merged into main. @MarcoEidinger Do you think it is possilble to create a branch just for designer review? We might need to setup TestFlight for this branch so that we can distribute test app to design team for early review even the development is still in progress. @dyongxu Any thoughts?

**Comment by MarcoEidinger** [comment]
> @MarcoEidinger Do you think it is possilble to create a branch just for designer review? We might need to setup TestFlight for this branch so that we can distribute test app to design team for early review even the development is still in progress.

@billzhou0223 feel free to merge into `migration` or setup a new branch.

If you are talking about the test app within this repository then deploying to TestFlight via GitHub Actions is an option as described in this [How-To](https://preece.dev/2020/06/ios-testflight-deploy-with-github-actions/) (not setup).

**Comment by MarcoEidinger** [comment]
FYI: merging should be squashed as this PR does not contain multiple new features or multiple fixes for released functionality. Have a single commit following conventional naming spec (`feat: ...`) as this will lead to proper CHANGELOG generation (once we trigger to publish a new release on GitHub)

**Comment by billzhou0223** [comment]
@fredsap Please fix conflicts and hond comments.

**Comment by fredsap** [comment]
all hound bot fixed

File: Apps/Apps.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>IDEDidComputeMac32BitWarning</key>
```
Comment (justinguo) [inline Apps/Apps.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:5]: Could you double check for the changes you made for `IDEWorkspaceChecks.plist`?

File: Apps/Examples/Examples/FioriSwiftUICore/SignatureView/SignatureView.swift [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+//  Examples
+//
+//  Created by Wirjo, Fred on 11/4/20.
+//  Copyright © 2020 SAP. All rights reserved.
+//
+
+import SwiftUI
+import FioriCharts
+import FioriSwiftUICore
+import Combine
```
Comment (justinguo) [inline Apps/Examples/Examples/FioriSwiftUICore/SignatureView/SignatureView.swift:12]: For the imports, could you check if everything is necessary? It seems like `FioriCharts` and `Combine` are unrelated.

**Comment by billzhou0223** [comment]
This PR is replaced by #212

**Pull request closed without merging**